presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -130,19 +129,6 @@ public QueryManagerConfig setMaxQueuedQueries(int maxQueuedQueries)
         return this;
     }
 
-    @Min(1)
-    public int getMaxPendingSplitsPerNode()
-    {
-        return maxPendingSplitsPerNode;
-    }
-
-    @Config("query.max-pending-splits-per-node")
```

Comment:
Add `@DefunctConfig` with this name on the class, so anyone using it will get a better error message.